app/main.py

```diff
@@ -1,2 +1,60 @@
+from typing import Any
+
+
+class Node:
+    def __init__(self, key: Any, value: Any) -> None:
+        self.key = key
+        self.value = value
+        self.hash = hash(key)
+
+
 class Dictionary:
-    pass
+    def __init__(self) -> None:
+        self.capacity = 8
+        self.load_factor = 2 / 3
```

Comment:
The load factor calculation '2 / 3' results in an integer division in Python 2, but in Python 3, it correctly results in a float. Ensure that this is intended, as the load factor should be a float to correctly compare against 'self.size / self.capacity'.

Reply:
Not valid